lib/internal/process/per_thread.js

```diff
@@ -149,7 +149,7 @@ function wrapProcessMethods(binding) {
       process._exiting = true;
       process.emit('exit', process.exitCode || 0);
     }
-    binding.reallyExit(process.exitCode || 0);
+    process.reallyExit(process.exitCode || 0);
```

Comment:
Can you add a comment for me here?

```suggestion
   // FIXME(joyeecheung): This is an undocumented API that gets monkey-patched
   // in the user land. Either document it, or deprecate it in favor of a better
   // public alternative.
    process.reallyExit(process.exitCode || 0);
```